crates/rpc/ipc/src/server/mod.rs

```diff
@@ -142,37 +163,27 @@ where
         let mut id: u32 = 0;
         let connection_guard = ConnectionGuard::new(self.cfg.max_connections as usize);
 
-        let mut connections = FutureDriver::default();
-        let endpoint_path = self.endpoint.path().to_string();
-        let incoming = match self.endpoint.incoming() {
-            Ok(connections) => {
-                #[cfg(windows)]
-                    let connections = Box::pin(connections);
-                Incoming::new(connections)
-            }
-            Err(err) => {
-                on_ready
-                    .send(Err(IpcServerStartError { endpoint: endpoint_path, source: err }))
-                    .ok();
-                return
-            }
-        };
-        // signal that we're ready to accept connections
-        on_ready.send(Ok(())).ok();
-
-        let mut incoming = Monitored::new(incoming, &stop_handle);
+        let mut connections = FuturesUnordered::new();
```

Comment:
jsonrpsee has switch to use FuturesUnordered in PR https://github.com/paritytech/jsonrpsee/pull/1062, which address some issues when upgrading tokio. The PR also removed the need of Incoming, Monitored, which unblock this PR.